docs/core-concepts/security/security-model.md

```diff
@@ -0,0 +1,200 @@
+- `LivenessThreshold` - The liveness threshold refers to the minimum percentage of total stake an attacker must control to cause a liveness failure.
+
+### Assumptions
+
+1. To guarantee **safety**, we assume that the adversary controls less than the `SafetyThreshold` percentage of the total stake.
+2. To guarantee **liveness**, we currently rely on a trusted disperser that does not censor clients’ blob disperser requests. We will soon introduce decentralized dispersal to remove this trust assumption. Additionally, to ensure liveness, we assume the adversary is delegated with less than `LivenessThreshold` percentage of the total stake.
+
+### Chunk Assignment Logic
+
+**Overview** 
+
+The chunk assignment logic assigns encoded chunks to validators in such a way that the security properties specified by the `SecurityParameters` can be verified.
+
+The chunk assignment logic provides the following primitives: 
+
+- `GetChunkAssignments`. Given a blob version and the state of the validators, generates a mapping from each chunk index to a validator.
+- `VerifySecurityParameters`. Validates whether a given set of security parameters is valid with respect to a blob version.
+
+For the purposes of modeling, we let $m_i$ denote the number of chunks which the assignment logic maps to an operator $i$. We also denote the `NumChunks` defined by the blob version as $m$, the `CodingRate` as $r$, and the `MaxNumOperators` as $n$. Any set of $m/r$ unique chunks can be used to recover the blob. We let $\alpha_i = rm_i/m$, which is the number of chunks assigned to validator $i$, divided by the number of chunks needed for recover the blob. We also denote $\eta_i$ as the percentage of quorum stake which is assigned to the validator $i$. The minimum percentage of the total stake that a group of validators must collectively hold in order to possess enough chunks to recover the blob is denoted by $\gamma$ . The confirmation threshold, safety threshold, and liveness threshold are denoted as $\eta_C$, $\eta_S$ and $\eta_L$. Additionally, $\eta_S = \eta_C - \gamma$ and $\eta_L = 1 - \eta_C$. The key terminology is summarized below for reference:
+
+| **Term** | **Symbol** | **Description** |
+| --- | --- | --- |
+| Max Validator Count |  $n$ | The maximum number of validator nodes participating in the system (currently $n =200$) |
+| Validator Set | $N$ | Set of all the validators. $|N|$ is the total number of validator nodes participating in the system. |
+| Total Chunks | $m$ | The total number of chunks after encoding (currently $m=8192$) |
+| Coding Rate | $r$ | The total number of chunks after encoding / total number of chunks before encoding (currently $r=8$) |
+| Percentage of blob per validator  | $\alpha_i$ | $\alpha_i = rm_i/m$, the percentage of chunks required to reconstruct the blob assigned to validator $i$ |
```

Comment:
It seems that this does not parse correctly in the github view
![image](https://github.com/user-attachments/assets/e1e19ea0-d564-4f58-ade3-9e01347f4eee)